### PR TITLE
Feature/65/update formatting

### LINKED ISSUE #65: Refactor code formatting feature to remove "feedback" step
**Is your feature request related to a problem? Please describe.**
Currently when adding the :robot: emoji on a message the bot tries to format the given code block using hightlightjs and prettier. If highlighjs gives 2 guesses back, the bot asks for clarification with 2 suggestions. This is slightly annoying and makes the feature harder to use, when most of the time the first guess is "right" anyways.

**Describe the solution you'd like**
Remove the feedback step so the bot just uses the first guess it has, and continues to format/setup the correct highlighted language just as before.

**Additional context**
This should be done **after** the migration to TS, see #55

### LINKED ISSUE #66: Add "format" command
**Is your feature request related to a problem? Please describe.**
Due to changes with the "reaction" code format feature, the bot will be more aggressive on auto-guessing the language. To provide a more "end-user" version that will provide more support for different languages and potential prettier support

**Describe the solution you'd like**
Add a new command `<prefix> format <message_link> <language>` command that will take a given message contents, the code in that message and try to format+apply syntax highlighting to that message.

**Describe alternatives you've considered**
This can use some of the existing utilities for the :robot: code format feature.

**Additional context**
Prettier should only be applied if the language given is actually recognized by prettier, otherwise don't try to format the code.

### LINKED ISSUE #67: Add specific case for HTML+JS language format guesses to format for XML
**Is your feature request related to a problem? Please describe.**
a code block like the following:
```xml
<script>
let me = "Awesome"
let you = "Better"
</script>
<div id="green>
<p id="blue>
</p>
</div>
```
can be formatted using XML. This can be used **if** the language guesses provided by highlightjs are HTML and JS (in that order). 

**Describe the solution you'd like**
See above, this would make JS in HTML syntax highlighting better for :robot: formatted code.

### PR DESCRIPTION
**Pull request by naomi-lgbt**
- [X] Removed the confirmation message - command now runs with the first guess automatically. 
- [X] Configured the command to use XML if it detects HTML and either CSS or JS.
- [X] Created a `format` command that takes the message ID as the parameter, and returns a formatted message. 

Closes #65, closes #66, and closes #67

@Twaha-Rahman - You wrote this feature originally, I believe. Wouldn't mind your eyes on this change too. 🙂

### CODE REVIEW

File: src/commands/format.ts
Comment (bradtaniguchi) [inline src/commands/format.ts]: Add try/catch block around this to log/capture any weird errors.

File: src/commands/format.ts
Comment (bradtaniguchi) [inline src/commands/format.ts]: Verify the amount is as expected, as if the content passed isn't long enough this code will explode.

File: src/commands/add-formatting.ts
Comment (bradtaniguchi) [inline src/commands/add-formatting.ts]: Use early return in the first `if` statement rather than an `else` statement. (Sorry I don't like else ;D)

File: src/commands/format.ts
Comment (naomi-lgbt) [reply src/commands/format.ts]: Done

File: src/commands/format.ts
Comment (naomi-lgbt) [reply src/commands/format.ts]: Added a check for length

File: src/commands/add-formatting.ts
Comment (naomi-lgbt) [reply src/commands/add-formatting.ts]: I like else :P
Done! :)

File: src/commands/format.ts
Comment (bradtaniguchi) [inline src/commands/format.ts]: Where and how is the user suppose to get the messageId? My original understanding was that this function would load via a discord URL?

Comment (naomi-lgbt) [reply src/commands/format.ts]: So...
That was the original plan.
But discord.js doesn't have a way to find a message by the url, that I could see.
So I went with this route instead.
But just now I've realised how I could parse the ID *from* the URL, and I'll go ahead and try to factor that into the command instead. Allow me a few minutes and I'll have this changed. 

Also, the ID can be copied just like the url (right-click drop down) if you've got dev mode turned on 🙂

Comment (naomi-lgbt) [reply src/commands/format.ts]: This is changed now.

**Review by bradtaniguchi — APPROVED** [review]
LGTM :+1: 
I'll wait for @Twaha-Rahman for a big if they want to review these changes aswell.

**Comment by bradtaniguchi** [comment]
I think its fine to merge this for now without any extra review from Twaha-Rahman

**Pull request merged**
